docs/mp/reactivemessaging/weblogic.adoc

```diff
@@ -34,18 +38,24 @@ include::{rootdir}/includes/mp.adoc[]
 
 == Overview
 
-Connecting streams to WebLogic JMS with Reactive Messaging couldn't be easier.
-This connector extends Helidon JMS connector with special handling for WebLogic T3 thin client.
+WebLogic JMS Connector extends Helidon JMS connector with special handling for legacy WebLogic T3 thin clients.
+Legacy versions of thin client can be found in server/lib directory(`WL_HOME/server/lib/wlthint3client.jar`) of any WebLogic Server installation.
 
-Connecting to WebLogic JMS connection factories requires proprietary T3 thin client library which can be obtained from
-WebLogic installation.
+Helidon supports {jakarta-version} and legacy versions of `javax` based thin T3 client wouldn't work correctly when on it's classpath.
```

Comment:
Helidon supports {jakarta-verion}, Legacy versions of `javax`-based thin T3 clients will not work when added to the classpath. Legacy thin T3 clients must be loaded from a filesystem location specified by the `thin-jar` property.